bin/ingestion.js

```diff
@@ -225,6 +202,61 @@ function updateProcessors(zkClient, bootstrapList) {
     });
 }
 
+/**
+ * Refresh processors by checking zookeeper states and use the state(s) to,
+ * update ingestionPopulator's paused locations list.
+ * @param {node-zookeeper-client.Client} zkClient - zookeeper client
+ * @param {Logger.newRequestLogger} log - request logger object
+ * @param {Function} cb - callback(error)
+ * @return {undefined}
+ */
+function refreshProcessors(zkClient, log, cb) {
+    const bootstrapList = config.getBootstrapList();
+    const active = Object.keys(configuredLocations);
+    const update = bootstrapList.map(i => i.site);
+    const locations = [...new Set(active.concat(update))];
+    const ingestionZkPath = getIngestionZkPath();
+
+    return async.each(locations, (location, next) => {
+        const path = `${ingestionZkPath}/${location}`;
+        return zkClient.getData(path, (err, data) => {
+            if (err) {
+                log.fatal('could not check location status in zookeeper',
+                    { method: 'bin::ingestion.refreshProcessors',
```

Comment:
Doesn't match the other formatting where the opening curly brace is on the previous line and the end curly is on a line by itself.

Reply:
Ah..my format plugin did not work in the editor..

Reply:
Fixed it. Also, it wont be uniform at all times because max line length of 80 is followed.

Reply:
Gotcha, makes sense. I like line limits ( although 80 is a bit low with 4 space tabs imo but meh ).